worldedit-core/src/main/java/com/sk89q/worldedit/extent/Extent.java

```diff
@@ -148,6 +149,20 @@ default Entity createEntity(Location location, BaseEntity entity) {
     }
 
     //FAWE start
+    /**
+     * Create an entity at the given location, forcing a UUID onto the entity
+     *
+     * @param entity   the entity
+     * @param location the location
+     * @param uuid     UUID to force the entity to have
+     * @return a reference to the created entity, or null if the entity could not be created
+     * @since TODO
+     */
+    @Nullable
+    @AnnotationHelper.ApiDescription(info = "Only use if you are aware of the consequences of forcing a UUID to an entity.")
```

Comment:
[-0.9](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions) on that one. This is P2 code  and it feels a bit weird to me to use that one here.